src/App.asm

```diff
@@ -1,6 +1,77 @@
 .data
 
+prompt1: .asciiz " Enter total number of entries: "
+
+prompt2: .asciiz " Enter an integer: "
+
+positiveMsg: .asciiz "\nTotal positive "
+
+negativeMsg: .asciiz "\nTotal negative "
 
 .globl main
 
-.text
\ No newline at end of file
+.text
+
+main: 
+
+	li $v0, 4 
+ 	la $a0, prompt1
+ 	syscall 
+
+
+ 	li $v0, 5 
+ 	syscall 
+
+
+ 	move $t0, $v0 
+
+
+	 li $t2, 0 
+
+	 li $t3, 0 
+
+ 	li $t4, 0 
+
+ 	li $t5, 0 
+
+    loop: 
+    
+     bge $t2,$t0,endLoop # if i >= count then go to endLoop
+
+
+     li $v0, 4 
+     la $a0, prompt2 
+     syscall 
+
+     li $v0, 5 
+     syscall 
+
+     move $t5,$v0   
+
+     bgtz $t5 ,positiveCount  
```

Comment:
you handle only positive numbers, and you count zero with negative number which is not correct (logical error)